EpikCord/__init__.py

```diff
@@ -1722,6 +1730,22 @@ def register_slash_command(func):
             }
         return register_slash_command
 
+    def get_commands(self) -> List[Union[ClientSlashCommand,ClientUserCommand,ClientMessageCommand]]:
+
+        return [c for c in self.commands]
+
+    def command_unload(self, name:str):
```

Comment:
This makes no sense

Reply:
what? there are two funcs

Reply:
How do you plan on loading the command back in once you pop it? Why are you raising a `SectionNotFound` error when the command is not valid?

Reply:
easy
put it in a separate dict
and put it back in reload function

Reply:
Just add a `Command.disabled` attribute and then you can do `Command.disable/enable()`? Probably more memory efficient as you are just setting an attribute in contrast to having to having an entirely separate `dict` filled with `Command`s